src/components/AddressSearch/index.tsx

```diff
@@ -343,14 +376,21 @@ function AddressSearch(
                         listEmptyComponent={listEmptyComponent}
                         listLoaderComponent={listLoader}
                         renderHeaderComponent={renderHeaderComponent}
-                        renderRow={(data) => {
+                        renderRow={(data, index) => {
                             const title = data.isPredefinedPlace ? data.name : data.structured_formatting.main_text;
                             const subtitle = data.isPredefinedPlace ? data.description : data.structured_formatting.secondary_text;
+                            if (focusedIndex === index) {
+                                setFocusedIndex(index);
+                            }
                             return (
-                                <View>
-                                    {title && <Text style={[styles.googleSearchText]}>{title}</Text>}
-                                    <Text style={[styles.textLabelSupporting]}>{subtitle}</Text>
-                                </View>
+                                <Hoverable>
+                                    {(hovered) => (
+                                        <View style={[styles.pv4, styles.ph3, StyleUtils.getBackgroundAndBorderStyle(hovered || focusedIndex === index ? theme.border : theme.appBG)]}>
```

Comment:
**question:** Why did move the padding from where it was to this view?

Reply:
the selected row was only highlighting the title of the address and the padding region outside the title was not getting highlighted. After removing the padding, the padded region outside the title was also getting highlighted on arrow key selection.